src/os/bluestore/BlueStore.cc

```diff
@@ -25,6 +25,7 @@
 
 #include "BlueStore.h"
 #include "bluestore_common.h"
+#include "simple_bitmap.h"
```

Comment:
Isn't mempool_dynamic_bitset sufficient?

Reply:
agree. Indeed boost:mempool_d* is missing some functionality, but you may want consider adding that on top of the existing library.